rasa/utils/common.py

```diff
@@ -319,3 +320,23 @@ async def call_potential_coroutine(
         return await coroutine_or_return_value
 
     return coroutine_or_return_value
+
+
+def copy_directory(source: Path, destination: Path) -> None:
+    """Copies the content of one directory into another.
+
+    Unlike `shutil.copytree` this doesn't raise if `destination` already exists.
+
+    # TODO: Drop this in favor of `shutil.copytree(..., dirs_exist_ok=True)` when
+    # dropping Python 3.7.
+
+    Args:
+        source: The directory whose contents should be copied to `destination`.
+        destination: The directory which should contain the content `source` in the end.
+
+    """
+    for item in source.glob("*"):
```

Comment:
not using `from distutils.dir_util import copy_tree` as it's not a recommended thing to do (see [here](https://bugs.python.org/issue10948))